compass/ocean/tests/sphere_transport/divergent_2d/analysis.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+from compass.step import Step
+from ..process_output import *
+from netCDF4 import Dataset
+
+
+class Analysis(Step):
+    """
+    A step for visualizing the output from the divergent2D test case
+
+    Attributes
+    ----------
+    resolutions : list of int
+        The resolutions of the meshes that have been run
+    """
+
+    def __init__(self, test_case, resolutions):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case :
+            compass.ocean.tests.sphere_transport.divergent_2d.divergent2D
```

Comment:
Same comment about line length -- this needs to go back to being one line.
And again, the description got chopped in the last commit